documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -312,6 +313,36 @@ For more information see Redis https://redis.io/commands/wait/[WAIT] command.
 |`1000`
 |If using Redis to store schema history, defines the delay of retry on wait for replica failure.
 
+
+|[[schema-history-internal-rocketmq-topic]]<<schema-history-internal-rocketmq-topic, `debezium.source.schema.history.internal.rocketmq.topic`>>
+|
+|If using RocketMQ to store schema history, you need to configure RocketMQ storage topic.
```

Comment:
I'd remove `If using RocketMQ to store schema history, you need to` and use just `The RocketMQ storage topic where history will be stored` or simply use same `withDescription()` strings from `RocketMqSchemaHistory`. Similar to other config items.